app/models/manageiq/providers/amazon/refresh_helper_methods.rb

```diff
@@ -41,6 +41,17 @@ def add_instance_disk(disks, size, name, location)
     super(disks, size, name, location, "amazon")
   end
 
+  def add_block_device_disk(disks, name, location)
+    disk = {
+      :device_name     => name,
+      :device_type     => "disk",
+      :controller_type => "amazon",
+      :location        => location,
+    }
+    disks << disk
```

Comment:
@gberginc - 
- will the block devices always be EBS volumes and not an instance store volume? Can this be reflected in the ```device_type``` attribute assuming any string value will do.
- Is there any more information available here eg size, free space? 
- Should this link back to the EBS volume object?